modules/appeals_api/spec/services/appeals_api/pdf_construction/generator_spec.rb

```diff
@@ -276,6 +276,8 @@
           allow_any_instance_of(AppealsApi::PdfConstruction::SupplementalClaim::V2::FormData).to receive(:signing_appellant_email).and_return('W' * 255)
 
           sc.form_data = data
+          # TODO update countryCodeISO2 in expected_200995_maxlength.pdf with expected override_max_lengths tranforms
+          sc.form_data['data']['attributes']['claimant']['address']['countryCodeISO2'] = 'WW' 
```

Comment:
this was a consequence of removing the min/max lengths on `countryCodeISO2` in the address schema. however, if we kept that in place, it would have just broken in the the test at line 212 instead for the inverse reason (i.e., transforming values to 'WW' when our expected PDFs still have them as different values)